openstf/model/reporter.py

```diff
@@ -1,50 +1,58 @@
+# SPDX-FileCopyrightText: 2017-2021 Alliander N.V. <korte.termijn.prognoses@alliander.com> # noqa E501>
+#
+# SPDX-License-Identifier: MPL-2.0
 import os
```

Comment:
Can you explain why the `reporter` should be part of the `model` package? Didn't we decide on the `metrics` package for reporting?

Reply:
moved the file

Reply:
If the comment is resolved, its probably a good idea to resolve it. Or you think I should do that?